### DISCUSSION TITLE
Flip Flopping between PR title formats

### DISCUSSION THREAD

**Discussion by mscrivo** [renovatebot/renovate#24441] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

For a long time now, renovate has been constantly updating PR's with title changes back and forth between:

Update dependency <dep> [npm]
fix(deps): update dependency <dep> [npm]

and it's causing a lot of noise in the PRs it creates.

Here's a sample:

<img width="530" alt="image" src="https://github.com/renovatebot/renovate/assets/275524/bef56e9d-e2cc-47db-ab9e-c8a1ec1b3c00">

### What you tried so far.

Nothing really, given this doesn't really seem to be a config issue? We don't have anything in our config that changes the title or branch name or anything like that. Just some standard packageRules to group certain dependencies.

### Relevant debug logs

Only thing I see in the logs is things like this:

<details><summary>Logs</summary>
DEBUG: PR title changed (branch="renovate/react-query-replacement")
{
  "branchName": "renovate/react-query-replacement"
  "oldPrTitle": "fix(deps): replace dependency react-query with @tanstack/react-query ^4.0.5 [npm]"
  "newPrTitle": "Replace dependency react-query with @tanstack/react-query ^4.0.5 [npm]"
}
</details>

**Comment by viceice** [comment, marked as answer]
you're mixing semver commit with none once. set semantic memory commit config option explicit

**Reply by rarkins** [reply]
https://docs.renovatebot.com/semantic-commits/

**Reply by mscrivo** [reply]
I think that did it, thank you both!
